src/run_build.py

```diff
@@ -70,10 +71,15 @@ def main() -> int:
                 logging.info(f"Successfully built {component.name}")
             except:
                 logging.error(f"Error building {component.name}, retry with: {args.component_command(component.name)}")
-                raise
+                if args.continue_on_error and component.name not in ['OpenSearch', 'OpenSearch Dashboards']:
+                    failed_plugins.append(component.name)
+                    continue
+                else:
+                    raise
 
         build_recorder.write_manifest()
-
+    if len(failed_plugins) > 0:
+        logging.error(f"Failed plugins are {failed_plugins}")
```

Comment:
If we error this out the exit code will be a non-zero value right? We still want the build to pass (with exit code as 0). I believe this should be `logging.info` ?

Reply:
I would let the Python code to fail but use the Jenkins workflow to decide whether it should fail the run.

The python code define as `continue on error` means it is `error`, just keep going, but we should not treat it as success.

Reply:
Its difficult to mark something error-ed out as not an error on CI level. The user wanted to build everything in manifest, it did with some failing components. I understand its a partial failure but if we want hard failure we can always omit the `--continue-on-error` option.
WDYT @prudhvigodithi @dblock?

Reply:
It is just logging level and doesn't exit with code 1 but 0. 
Agree with @gaiksaya , if you want to fail the build on component failure then skip the optional flag.

Reply:
Even with `continue-on-error` we will be logging the exception message which will make sure our github auto-create issue workflow would still cut issues for failing components.
The failing components will be logged.  @bbarani